### PR TITLE
Add Baiducloud conformance test into testgrid

### PR DESCRIPTION
**Pull request by tizhou86**
Hi all, we'd like to add baiducloud conformance test into testgrid, I've just pushed our test result into the gcs bucket, if there's something wrong, please let me know, thanks!

### CODE REVIEW

**Comment by tizhou86** [comment]
#16587

**Comment by tizhou86** [comment]
Hi @BenTheElder, any comments on this PR? Thanks!

File: config/testgrids/conformance/conformance-all.yaml [hunk 1]
```diff
@@ -219,6 +220,12 @@ dashboards:
       description: Runs conformance tests using kubetest against kubernetes v1.17 with cloud-provider-huaweicloud
       test_group_name: cloud-provider-huaweicloud-e2e-conformance-release-v1.17
 
+- name: conformance-cloud-provider-baiducloud
+  dashboard_tab:
+    - name: Baidu Cloud Provider, v1.13
```
Comment (BenTheElder) [inline config/testgrids/conformance/conformance-all.yaml]: I'm not going to block this right now as I don't think we have policy, but 1.13 is way out of support upstream and cannot be certified for conformance currently

Comment (BenTheElder) [reply config/testgrids/conformance/conformance-all.yaml]: we were really mostly hoping for people to publish tests against the latest kubernetes code when we set up the program for people to publish conformance results to testgrid :disappointed:

**Comment by BenTheElder** [comment]
going to let @spiffxp decide this

**Comment by spiffxp** [comment]
/approve
/lgtm
I won't hold this up for now, because there are already many other tabs for kubernetes <=1.13.  But I agree we may consider a policy of pruning off older releases, so it would be preferable to upload results for something more current

**Pull request merged**